plugins/systemd/systemd.plugin.zsh

```diff
@@ -20,3 +20,14 @@ alias sc-mask-now="sc-mask --now"
 alias scu-enable-now="scu-enable --now"
 alias scu-disable-now="scu-disable --now"
 alias scu-mask-now="scu-mask --now"
+
+function sc-prompt-status {
```

Comment:
For consistency with other prompt functions, use `systemd_prompt_info` instead. See `lib/prompt_info_functions.zsh`